### PR TITLE
Updating steps for way number 10

### LINKED ISSUE #2193: Instructions do not match UI
On section 10: Protect against phishing attacks with ATP Safe **Links**, the steps described do not match the UI.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ea79921a-7f83-58f1-c0a4-6eb0efce0d26
* Version Independent ID: b60110ef-71ad-9962-74de-734062708ff8
* Content: [Top 10 ways to secure Microsoft 365 for business plans - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/security-and-compliance/secure-your-business-data?view=o365-worldwide#malware)
* Content Source: [microsoft-365/admin/security-and-compliance/secure-your-business-data.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/security-and-compliance/secure-your-business-data.md)
* Service: **o365-administration**
* GitHub Login: @Sirkkuw
* Microsoft Alias: **sirkkuw**

### PR DESCRIPTION
**Pull request by get-itips**
@Sirkkuw please review
cc @mypil 
closes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2193

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review on the additional commits of this PR. Thanks.

File: microsoft-365/admin/security-and-compliance/secure-your-business-data.md [hunk 1]
```diff
@@ -321,15 +321,17 @@ To get to ATP Safe Links, view a [short training video](https://support.microsof
     
 To modify the default policy:
   
-1. On the Safe links page, under **Policies that apply to the entire organization**, select the **Default** policy. 
+1. On the Safe links page, under **Policies that apply to the entire organization**, double-click the **Default** policy. 
     
-2. Under **Settings that apply to content except email**, select **Microsoft 365 Apps for enterprise, Office for iOS and Android**.
+2. Under **Settings that apply to content across Office 365**, enter an URL to be blocked, such as _example.com_ and select **+**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
2. Under **Settings that apply to content across Office 365**, enter a URL to be blocked, such as _example.com_, and select **+**.
```

File: microsoft-365/admin/security-and-compliance/secure-your-business-data.md [hunk 2]
```diff
@@ -321,15 +321,17 @@ To get to ATP Safe Links, view a [short training video](https://support.microsof
...
+
+3. Under **Settings that apply to content except email**, select **Office 365 applications**, **Do not track when users click safe links** and **Do not let users click through safe links to original URL**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
3. Under **Settings that apply to content except email**, select **Office 365 applications**, **Do not track when users click safe links**, and **Do not let users click through safe links to original URL**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #2193.

Thank you.

**Pull request merged**